netplugin/cluster/cluster.go

```diff
@@ -230,7 +230,7 @@ func masterReq(path string, req interface{}, resp interface{}, isDel bool) error
 	}
 
 	log.Errorf("Error making POST request. All master failed")
-	return errors.New("POST request failed")
+	return errors.New("the POST request failed")
```

Comment:
In general if you are changing a string, 
can you make it to more descriptive like "http POST request to master failed "